### DISCUSSION TITLE
how to make a bottle in a usb?

### DISCUSSION THREAD

**Discussion by arsolana0** [bottlesdevs/Bottles#558] [Q&A]
so i want to install a pretty big application and i dont want it on my pc. is there a way to make a bottle in a custom install directory or something like that?

**Comment by mirkobrombin** [comment]
Support for custom path is planned but currently not in development. As a workaround, create the new bottle, then move its content in the new path and hard-link to the old one. Make sure to give Bottles the permissions if using the Flatpak (using [Flatseal](https://docs.usebottles.com/flatpak/expose-directories)).
